SUMMARY.md

```diff
@@ -9,24 +9,24 @@
 ## Hardware
 
 * [Components](hardware/components.md)
-    * [Topside](hardware/components.md#topside-components)
```

Comment:
Why do we have these diffs

Reply:
In my first comment I said that gitbook-editor changed this, maybe a default indentation level for md ? IDK.

Reply:
FIne, it will just happen again if we revert then.